compiler/cmd/check-backward-compatibility/check-backward-compatibility.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
```

Comment:
@PavitraS why this renaming ? We are still generating openapi schema correct ?
Are these are two different activities: 
1. generation openapi schema 
2. running comp check on the generated schema

Reply:
Reverted the change.